actionview/test/template/testing/fixture_resolver_test.rb

```diff
@@ -17,4 +17,14 @@ def test_should_return_template_for_declared_path
     assert_equal "arbitrary/path", templates.first.virtual_path
     assert_equal [:html],          templates.first.formats
   end
+
+  def test_should_match_templates_with_variants
+    resolver = ActionView::FixtureResolver.new("arbitrary/path.html+variant.erb" => "this text")
+    templates = resolver.find_all("path", "arbitrary", false, locale: [], formats: [:html], variants: [:variant], handlers: [:erb])
+    assert_equal 1, templates.size, "expected one template"
+    assert_equal "this text",       templates.first.source
+    assert_equal "arbitrary/path",  templates.first.virtual_path
+    assert_equal [:html],           templates.first.formats
+    assert_equal ["variant"],          templates.first.variants
```

Comment:
The indentation here is off

Reply:
@rafaelfranca fixed (and I would nitpick on that too:) ).

let me know when/if you want me to rebase+squash